website/docs/foundations/icons/partials/engineering-guidelines.md

```diff
@@ -186,29 +204,31 @@ import { IconArrowRight24 } from '@hashicorp/flight-icons/svg-react/arrow-right-
 <IconArrowRight24 />
 ```
 
-_Notice: the code above is an example, please update it accordingly to your codebase._
+!!! Information
+
+The code above is an example; please update it accordingly to your codebase.
+!!!
 
 #### Props
 
 The component exposes the following _props_:
 
-1.  `color` - the color (applied as _fill_) to the SVG - by default is `currentColor` but any valid HTML/CSS color is accepted
-2.  `title` - the title of the SVG - by default the icon has an _aria-hidden_ attribute applied to it, because is expected to be used _in contex_ (see [§ Accessibility](#accessibility)); if instead you need to use it without text associated to it, you have to pass a _title_ attribute to make it accessible.
-3.  `...props` - any other _prop_ passed to the component will be applied via spread
+1.  `color` - the color (applied as _fill_) to the SVG - by default is `currentColor` but any valid HTML/CSS color is accepted.
+2.  `title` - the title of the SVG - by default, the icon has an _aria-hidden_ attribute applied to it because it is expected to be used in context (check out [§ Accessibility](#accessibility)); if instead you need to use it without text associated to it, you have to pass a _title_ attribute to make it accessible.
```

Comment:
```suggestion
2.  `title`—the title of the SVG—by default, the icon has an _aria-hidden_ attribute applied to it because it is expected to be used in context (check out [§ Accessibility](#accessibility)); if instead you need to use it without text associated to it, you have to pass a _title_ attribute to make it accessible.
```